pkg/dao/types/status/status_servicerevision.go

```diff
@@ -1,38 +1,31 @@
 package status
 
 const (
-	ServiceRevisionStatusReady ConditionType = "Ready"
+	ServiceRevisionStatusRunning ConditionType = "Running"
+	ServiceRevisionStatusReady   ConditionType = "Ready"
 
 	ServiceRevisionSummaryStatusRunning string = "Running"
 	ServiceRevisionSummaryStatusFailed  string = "Failed"
-	ServiceRevisionSummaryStatusSucceed string = "Succeed"
+	ServiceRevisionSummaryStatusSucceed string = "Ready"
```

Comment:
Can you explain the condition changes?

Reply:
refactor revision and deployer will not influence workflow feature, I will create another Pull Request.